cogs/reward.py

```diff
@@ -43,89 +45,118 @@ def convert_to_seconds(s):
-            else:
-                await channel.send(content=f"A {durationString} game would give **{charDict['Name']}** \n{treasureString}\n**{charDict['Name']}** will be level {int(resultLevel)} with {int(nextCP)} CP with an additional {treasureArray[1].pop(tierInitial)} T{t+1} TP, {treasureArray[1].pop(tierNext)} T{t+2} TP,  and {totalGold} gold total!")
+            if len(tpGained) == 1:  # Case for when the character stays in the tier
+                await channel.send(content=f"A {durationString} game would give **{charDict['Name']}** \n{treasureString}\n**{charDict['Name']}** will be level {int(resultLevel)} with {int(nextCP)} CP with an additional {treasureArray[1].pop(tierInitial)} T{tierNum+1} TP and {totalGold} gold total!")
+            elif len(tpGained) == 2: # Case for crossing just one tier
+                await channel.send(content=f"A {durationString} game would give **{charDict['Name']}** \n{treasureString}\n**{charDict['Name']}** will be level {int(resultLevel)} with {int(nextCP)} CP with an additional {treasureArray[1].pop(tierInitial)} T{tierNum+1} TP, {treasureArray[1].pop(tierNext)} T{tierNum+2} TP, and {totalGold} gold total!")
+            elif len(tpGained) == 3: # Rest are cases for cheeky bastards who think they'll be in absurdly long games
+                await channel.send(content=f"A {durationString} game would give **{charDict['Name']}** \n{treasureString}\n**{charDict['Name']}** will be level {int(resultLevel)} with {int(nextCP)} CP with an additional {treasureArray[1].pop(tierInitial)} T{tierNum+1} TP, {treasureArray[1].pop(tierNext)} T{tierNum+2} TP, {treasureArray[1].pop(tierNext1)} T{tierNum+3} TP, and {totalGold} gold total! Did you remember to hydrate?")
+            elif len(tpGained) == 4:
+                await channel.send(content=f"A {durationString} game would give **{charDict['Name']}** \n{treasureString}\n**{charDict['Name']}** will be level {int(resultLevel)} with {int(nextCP)} CP with an additional {treasureArray[1].pop(tierInitial)} T{tierNum+1} TP, {treasureArray[1].pop(tierNext)} T{tierNum+2} TP, {treasureArray[1].pop(tierNext1)} T{tierNum+3} TP, {treasureArray[1].pop(tierNext2)} T{tierNum+4} TP, and {totalGold} gold total! Please, check on the kids.")
+            elif len(tpGained) == 5:
+                await channel.send(content=f"A {durationString} game would give **{charDict['Name']}** \n{treasureString}\n**{charDict['Name']}** will be level {int(resultLevel)} with {int(nextCP)} CP with an additional {treasureArray[1].pop(tierInitial)} T{tierNum+1} TP, {treasureArray[1].pop(tierNext)} T{tierNum+2} TP, {treasureArray[1].pop(tierNext1)} T{tierNum+3} TP, {treasureArray[1].pop(tierNext2)} T{tierNum+4} TP, {treasureArray[1].pop(tierNext3)} T{tierNum+5} TP, and {totalGold} gold total! Of course, you will die of exhaustion from playing that much in one game, but yay tier 5???")
```

Comment:
Condense it so you don't need an if statement. 5 different statements is not acceptable